lib/elixir/src/elixir_interpolation.erl

```diff
@@ -151,8 +165,7 @@ unescape_hex(<<${, A, B, C, D, E, F, $}, Rest/binary>>, Map, Acc) when ?is_hex(A
   append_codepoint(Rest, Map, [A, B, C, D, E, F], Acc, 16);
 
 unescape_hex(<<_/binary>>, _Map, _Acc) ->
-  Msg = <<"missing hex sequence after \\x, expected \\xHH">>,
-  error('Elixir.ArgumentError':exception([{message, Msg}])).
+  throw(missing_hex_sequence).
```

Comment:
I was not sure what we would want to throw here and am not sure how we all feel about just throwing `missing_hex_sequence`. ❔

Reply:
This is totally fine but you can also do `throw({error, Msg})` and you catch and return the tuple later on.

Reply:
Brilliant! That sounds like the route we'd want. Let me update! 👍